packages/reporter/src/commands/command.cy.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import Command from './command'
+import CommandModel from './command-model'
+
+describe('commands', () => {
+  describe('sessionPill', () => {
+    const statusList = [
+      'creating',
+      'created',
+      'restoring',
+      'restored',
+      'recreating',
+      'recreated',
+      'failed',
+    ]
+
+    it('session status in command', () => {
+      cy.mount(
```

Comment:
Woo, component tests! Reporter is a great candidate for smaller, self contained components.

Reply:
I'd say most/ if not all all of the reporter tests could be converted to component tests at some point!